src/components/ContextSubMenu.js

```diff
@@ -9,12 +10,13 @@ function ContextSubMenu({heading, collection, directory}) {
         { collection.map(item => (
           <li
             key={item.node.parent.name}
-            className="docs-menu__collection-item">
-            <Link
-              className="docs-menu__link"
```

Comment:
can we remove the css for docs-menu__link?

Reply:
Yup. Looks like we don't use it anywhere else. I'll remove it.